test/unit/preprocessing/linear_dep_remover/test_sklearn_compat.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 import pandas as pd
+import polars as pl
```

Comment:
Should we test with pyarrow as well?

Reply:
I implemented different dataframe constructors similarly to how we do it in narwhals itself, but also how plotly now runs its test suite (see [plotly.py/tests/test_optional/test_px/conftest.py](https://github.com/plotly/plotly.py/blob/main/tests/test_optional/test_px/conftest.py))